### ISSUE TITLE
Missing radar_parameters and calibrations, and georeferencing in nexrad_level2

### ISSUE THREAD

**Issue by syedhamidali** [openradar/xradar#232]
* xradar version: 0.7+
* Python version:3.12
* Operating System:MacOS

### Description
I cant see radar parameters and georeferencing groups

### What I Did

```python
import xradar as xd
from open_radar_data import DATASETS

file = DATASETS.fetch("KLBB20160601_150025_V06")

tree = xd.io.open_nexradlevel2_datatree(file)

print(tree.groups)
```

```python
('/',
 '/sweep_0',
 '/sweep_1',
 '/sweep_2',
 '/sweep_3',
 '/sweep_4',
 '/sweep_5',
 '/sweep_6',
 '/sweep_7',
 '/sweep_8',
 '/sweep_9',
 '/sweep_10')
```

**Comment by aladinor** [comment]
I think this might resolved with #221.

**Comment by syedhamidali** [comment]
@aladinor Yes, I installed your data-constructor branch and gave it a try, it is working... Thanks!

**Comment by syedhamidali** [comment]
Solved in #221

**Comment by syedhamidali** [comment]
Although @aladinor’s changes resolved the issue with these groups, it turns out they are empty. I think the error lies in the nexrad_level2 reader.

**Comment by kmuehlbauer** [comment]
Currently there is no mapping between the nexrad metadata and those groups. Nexrad is not the only engine which would need a proper mapping. #221 was the first big step to make this possible at all by making these groups available (even if not filled).

We should keep this issue open until we've worked the different backends mapping.
